packages/RangeFilter/src/index.tsx

```diff
@@ -63,8 +63,11 @@ const RangeFilter: FunctionComponent<IRangeFilterProps> = ({
 
   useEffect(() => {
     const rangeFilter = init(filterRef.current);
-    rangeFilter.onChange = (minvalue: any, maxvalue: any) =>
-      onChange({ value: { min: minvalue, max: maxvalue } });
+    rangeFilter.onChange = (minvalue: any, maxvalue: any) => {
+      const newValue = { ...inputs, minValue: minvalue, maxValue: maxvalue };
+      onChange({ value: newValue });
+      setInputs(newValue);
+    };
```

Comment:
This looks like quiet strange, cannot u pass the onChange prop directly to the component?

Reply:
I have two onChange the first is a prop and the other is a RangeFilter event that returns the current position of the two controllers